.ci_support/migrations/cutensor2.yaml

```diff
@@ -3,7 +3,14 @@ __migrator:
   kind: version
   migration_number: 1
   bump_number: 1
-  use_local: True
+  commit_message: |
+    Rebuild for cuTENSOR 2.x support
+    
+    This migrator updates the dependency on `cutensor` to 2.x. Since 2.x has API
+    breaking changes, be sure your package has the latest support tested before
+    merging. Otherwise, please keep the PR open (and better, in draft status) as
+    instructed [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#migrators-and-migrations)
+    until the package is ready to migrate.
```

Comment:
Why did this change get snuck into here? If it's desirable, can we update the PR title and description to indicate that this was left out of a previous build it should have been in?

Reply:
Leo was dropping the `use_local: True` line above as this migrator was vendored in the feedstock (instead of using the global one for some reason)

By dropping `use_local: True` the global migrator is used (and this one ignored except for an existence check)

Leo also refreshed the contents of this migrator, which amounted to adding a commit message, but that doesn't change anything here as that only affects how bot PRs are titled and their OPs written (this matters even less with `use_local: True` dropped)

I guess we could update the PR to include this, but it doesn't change how CuPy builds are being done

Reply:
Indeed. It took me a while in https://github.com/conda-forge/cupy-feedstock/pull/241 to realize (with hindsight it's an obvious thing to do) that I needed a migrator to switch to use cuTENSOR 2.0. So I created one, used it in that PR, and [backported it](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5410) to the global pinning (again, with hindsight I could have just [let the bot take care of it](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5176)). Since we already set `use_local`, the bot would just skip this feedstock and marked the migration as done. So now that we're touching the code here, it makes sense for me to manually align and drop the `use_local` field. As John alluded to, if after a rerendering nothing is changed, it means we're safe and doing the right thing, since we've manually migrated in the first place.